installer/install.iss

```diff
@@ -3332,6 +3339,31 @@ begin
         InstallWindowsTerminalFragment();
     end;
 
+    {
+        Optionally "skip" installing bundled SSH binaries conflicting with Windows OpenSSH:
+        https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_overview
+    }
+
+    if RdbSSH[GS_WindowsOpenSSH].Checked then begin
+        WizardForm.StatusLabel.Caption:='Removing bundled Git OpenSSH binaries';
+        if not DeleteFile(AppDir+'\usr\bin\scp.exe') then
+            LogError('Line {#__LINE__}: Unable to delete "scp.exe".');
+        if not DeleteFile(AppDir+'\usr\bin\sftp.exe') then
+            LogError('Line {#__LINE__}: Unable to delete "sftp.exe".');
+        if not DeleteFile(AppDir+'\usr\bin\ssh-add.exe') then
+            LogError('Line {#__LINE__}: Unable to delete "ssh-add.exe".');
+        if not DeleteFile(AppDir+'\usr\bin\ssh-agent.exe') then
+            LogError('Line {#__LINE__}: Unable to delete "ssh-agent.exe".');
+        if not DeleteFile(AppDir+'\usr\bin\ssh-keygen.exe') then
+            LogError('Line {#__LINE__}: Unable to delete "ssh-keygen.exe".');
+        if not DeleteFile(AppDir+'\usr\bin\ssh-keyscan.exe') then
+            LogError('Line {#__LINE__}: Unable to delete "ssh-keyscan.exe".');
+        if not DeleteFile(AppDir+'\usr\bin\ssh.exe') then
+            LogError('Line {#__LINE__}: Unable to delete "ssh.exe".');
+        if not DeleteFile(AppDir+'\usr\bin\sshd.exe') then
```

Comment:
I fear that we will need to generate this list from the lines of `pacman -Ql openssh`'s output matching the output of `make-file-list.sh`, i.e. what is available in [the `$LIST` variable](https://github.com/git-for-windows/build-extra/blob/main/installer/release.sh#L158) in `please.sh`.

Every once in a while, a new  `.exe` enters the fray (and of course there is `/usr/lib/ssh/`), and doing it in the way I suggested does not risk `install.iss` going stale.

Reply:
I am wondering about that because we specifically need to match against stuff that's already on the `PATH` because someone else supplied it; right now I did it the dumb way and checked the Microsoft supplied list (double checking against my test system) and statically delete the files that should already exist.

I'll see whether I can come up with something but ultimately, yes this should be solved in a maintainable fashion. I am open to suggestions/code snippets, though, because Inno Setup related stuff really isn't my forte 😅.